### PR TITLE
RBAC Application  

### PR DESCRIPTION
**Pull request by auslin-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE
https://aottech.atlassian.net/browse/FWF-2287

# Changes
- new enum APPLICATION added in auth type
- change in application list endpoint
- change in the application by id endpoint
- application count in draft endpoint 
- migration script changes to move existing records.

# Screenshots
image
image

### CODE REVIEW

File: forms-flow-api/src/formsflow_api/models/authorization.py [hunk 1]
```diff
@@ -21,6 +21,7 @@ class AuthType(Enum):
     FORM = "FORM"
     FILTER = "FILTER"
     DESIGNER = "DESIGNER"
+    APPLICATION = "APPLICATION"
```
Comment (shuhaib-aot) [inline forms-flow-api/src/formsflow_api/models/authorization.py:24]: the better way to use these variables we can create an enumerator class ?

File: forms-flow-web/src/components/Form/List.js [hunk 2]
```diff
@@ -278,6 +278,8 @@ const List = React.memo((props) => {
     };
     addUsers(payload).catch((error) => console.error("error", error));
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/Form/List.js:279]: in future need to change function name 
addUsers -> addDesigners

**Review by shuhaib-aot — APPROVED** [review]

File: forms-flow-api/src/formsflow_api/models/authorization.py (hunk 1 above)
Comment (auslin-aot) [reply forms-flow-api/src/formsflow_api/models/authorization.py:24]: It is an enumeration class.
image

File: forms-flow-web/src/components/Form/List.js (hunk 2 above)
Comment (Bonymol-aot) [reply forms-flow-web/src/components/Form/List.js:279]: Noted

**Review by shuhaib-aot — APPROVED** [review]

**Review by Bonymol-aot — APPROVED** [review]

**Pull request merged**